src/pages/home/platform-carousel/_utils.tsx

```diff
@@ -95,7 +92,7 @@ export const platform_details_cr: TPlatformDetails[] = [
         description: <Localize translate_text="The CFD trading platform to fit your style." />,
         learn_more_link: '/derivx/',
         download_links: [
-            { type: 'browser', url: derivx_app_url },
+            { type: 'browser', link_type: 'deriv_app' },
```

Comment:
Just to clarify, we don't have `derivx_app` type for this one?

Reply:
We do! Mistaken that for Deriv app. Will change it, thanks for spotting it!